src/libraries/System.Private.Uri/src/System/UriHelper.cs

```diff
@@ -668,35 +668,36 @@ internal static void EscapeAsciiChar(byte b, ref ValueStringBuilder to)
             HexConverter.ToCharsBuffer(b, to.AppendSpan(2), 0, HexConverter.Casing.Upper);
         }
 
-        internal static char EscapedAscii(char digit, char next)
+        internal static char EscapedAscii(uint digit, uint next)
```

Comment:
Might be worth a comment indicating what this method does, and maybe a better name. Eg. 
```c#
// Converts 2 hex chars to a byte (returned in a char) 
// e.g, '0' 'a' becomes (char)0x0A
// If either char is not hex, returns (char)0xFFFF
internal static char UnescapeHexChars(uint digit, uint next)
```
Unless I'm misunderstanding.

Reply:
I've renamed it to `DecodeHexChars` and added a comment